cmd/truss/main.go

```diff
@@ -56,14 +55,18 @@ func main() {
 	genFiles, err := generateCode(cfg, dt, sd)
 	exitIfError(errors.Wrap(err, "cannot generate service"))
 
-	for _, f := range genFiles {
-		err := writeGenFile(f, cfg.ServicePath)
+	for path, file := range genFiles {
+		err := writeGenFile(file, path, cfg.ServicePath)
 		if err != nil {
 			exitIfError(errors.Wrap(err, "cannot to write output"))
 		}
 	}
 }
 
+// fileSystem is a map of paths relative to truss.Config.ServicePath mapped to
+// io.Reader's as files
```

Comment:
It's fewer characters, but it obfuscates its meaning. I have to know what a `fileSystem` is to read that function signature, whereas there's no ambiguity about what a `map[string]io.Reader` is. The Go way values explicitness and readability--where readability here is the ability to read the code and understand it without having to cross reference other things.

If you want to name the type, I would make it more plain rather than trying to abstract away its definition in the name. E.g., `readerMap`. But honestly, I feel like we don't need the type. It's not used so prolifically that the slight keystroke savings is worth the slight obfuscation.